books/RayTracingTheRestOfYourLife.html

```diff
@@ -110,23 +110,24 @@
 
     #include <iostream>
     #include <iomanip>
-    #include <math.h>
-    #include <stdlib.h>
 
     int main() {
-        int N = 100000;
+        std::cout << std::fixed << std::setprecision(12);
+
         int inside_circle = 0;
+        int N = 100000;
+
         for (int i = 0; i < N; i++) {
             auto x = random_double(-1,1);
             auto y = random_double(-1,1);
             if (x*x + y*y < 1)
                 inside_circle++;
         }
-        std::cout << std::fixed << std::setprecision(12);
+
         std::cout << "Estimate of Pi = " << (4.0 * inside_circle) / N << '\n';
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [estpi-1]: <kbd>[pi.cc]</kbd> Estimating π]
+    [Listing [estpi-1]: <kbd>[pi.cc]</kbd> Estimating π -- version one]
```

Comment:
I limit code styles and mathematical typesetting in the captions. If you think we should reconsider this, validate that Markdeep is cool with this (also use single quotes instead of HTML `<kbd>` tags), and create a new issue for the backlog.

Reply:
Looks like using back quotes gets the background-shaded and italic code style in the caption, which is no good. However, I believe this would be an easy adjustment to our CSS. MathJax looks fine. I'm fine with doing this work.

Reply:
Seems to work well with just $\pi$

<img width="745" alt="image" src="https://github.com/RayTracing/raytracing.github.io/assets/9646385/6b14fc46-04e5-40cb-b099-dae98723919e">

Reply:
Heh, we have captions with exponents that get rendered with superscripts, but likely due to Markdeep, and not MathJax. I'll have to revisit all captions (image and listing).

I'm going to let the filename sit for now with `<kbd>` tags, and will (sigh) go ahead and update the captions, since we're using math notation that's not properly rendered with either Markdeep or MathJax.

Reply:
Ah, we're already using MathJax in captions in book 2...

Reply:
Changes made.